ai_review/rules/security_rules.py

```
@@ -0,0 +1,22 @@
+def detect_unsafe_functions(tree) -> list:
+    """识别高危函数调用"""
+    unsafe_calls = {'eval', 'exec', 'pickle.loads'}
+    return [{
+        'msg': f"检测到不安全函数调用: {node.func.id}",
+        'level': 'critical',
+        'lineno': node.lineno
+    } for node in ast.walk(tree) 
```

Ruff (0.8.2)

Undefined name `ast`

(F821)